activities/staff-management/leave.md

```diff
@@ -31,6 +31,24 @@ More days can be bought at cost rate with agreement from your manager and <hr@pu
 
 The leave can now be approved by your manager.
 
+### Buying more vacation time or deduct from next year's quota
+
+Firstly, you can see how much leave you have when logging into your Tentoo NMBRS account. Navigate to the "leave" section on your dashboard.
+
+Please let the Operations Coordinator know if you want to take more vacation time than you have an if you want the additional leave to be deducted from next year's quota (Tentoo NMBRS does this automatically) or if you want to buy more days.
+
+When buying more vacaction time you need to [fill out a form approving the deduction from your salary](https://docs.google.com/document/d/1l-Z03ri4We4rv6inEvudjW0XYt2k0-p0vkCrhbb_1sw/edit). After the form is signed and received by the Operations Coordinator the amount will be deducted in the upcoming salary run, or the salary run of the period where the extra time off is taken.
+
+You may also want to know what extra vacation time actually costs:
+
+Take your “uurloon” hourly salary (which excludes vacation) from your payslip. Multiply that by 8 hours to get the gross day salary. Then "bijzonder tarief" is deducted (37,35 + 11,67 % = 49,02 %). “Bijzonder tarief, tarief BT or “special tax rate” is a % of “loonheffing” of the wage tax. The special wage tax rate is the tax rate that is applied for non-recurring remunerations, such as a one-time payment of unused vacation days. This rate also applies to this use case.
```

Comment:
Reading this, I wondered if the percentage was an error. I've updated the example below to make clear that the percentage is fully intentional.

Reply:
It's some weird payrolly thing, or just simply wage taxes being unnecessarily complicated - thanks!